libs/lb/lb-rs/src/io/docs.rs

```diff
@@ -22,6 +28,34 @@ pub struct AsyncDocs {
     location: PathBuf,
 }
 
+#[cfg(target_family = "wasm")]
```

Comment:
It should end up documented somewhere that lb-rs doesn't support wasm beyond a "demo" capacity. Not sure where, maybe in the config you define, see below comment.